lib/detect.js

```diff
@@ -0,0 +1,55 @@
+const Octokit = require('@octokit/rest');
```

Comment:
In the original repo, this file was an independent project which we had to publish to NPM if we wanted to use as a separate thing. I decided to merge all the code within a single project for ease of use.